package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "react-native-hockeyapp",
-  "version": "0.5.1",
+  "version": "0.6.1",
```

Comment:
We're going to set this to `0.1.0` for now since we don't have a plan to merge this back into the original repo yet. We'll need to get a better solution before doing that. This at least allows us to publish our own version in the future if we don't go back into the main.